src/include/reservation.h

```diff
@@ -63,6 +63,8 @@ extern "C" {
 
 #define RESV_START_TIME_MODIFIED	0x1
 #define RESV_END_TIME_MODIFIED		0x2
+#define RESV_DURATION_MODIFIED		0x3
```

Comment:
Where does this get used? And are you sure you can use 3 or is it a bit field? Might need to change it to 0x4.

Reply:
Yeah, this is a bitfield, it should be 0x4